ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss

```diff
@@ -51,6 +51,54 @@
     text-align: right;
     width: 10%;
```

Comment:
I think will be useful to extract styles also like

- edit-gas-tooltip
     - edit-gas-tooltip.js 
     - edit-gas-tooltip.test.js 
     - edit-gas-tooltip.scss

Reply:
👍